lib/view_model/exchange/exchange_view_model.dart

```diff
@@ -53,15 +56,18 @@ abstract class ExchangeViewModelBase with Store {
       isDepositAddressEnabled = false,
       isReceiveAddressEnabled = false,
       isReceiveAmountEditable = false,
+      _providerUseTorOnly = false,
       receiveCurrencies = <CryptoCurrency>[],
       depositCurrencies = <CryptoCurrency>[],
       limits = Limits(min: 0, max: 0),
       tradeState = ExchangeTradeStateInitial(),
       limitsState = LimitsInitialState(),
       receiveCurrency = wallet.currency,
       depositCurrency = wallet.currency,
-      providerList = [ChangeNowExchangeProvider(), SideShiftExchangeProvider(), SimpleSwapExchangeProvider()],
+      providerList = [],
       selectedProviders = ObservableList<ExchangeProvider>() {
+    _providerUseTorOnly = _settingsStore.exchangeStatus == FiatApiMode.torOnly;
```

Comment:
this should be `ExchangeApiMode.torOnly`
also rename the boolean to `_useTorOnly` better